doc/test_types.md

```diff
@@ -0,0 +1,54 @@
+# Types and levels of tests
+
+## Node System tests
+
+### Node Nightly CLI tests
+
+- All the CLI end-to-end tests are run automatically, on a nightly basis, on different feature combinations, using the latest `node master` branch
+- The results of the latest runs can be found [here](https://input-output-hk.github.io/cardano-node-tests/test_results.html)
+
+### Node CLI tests (on demand)
```

Comment:
maybe we could explain how the tests can be manually triggered 

![Screenshot from 2022-09-30 16-29-13](https://user-images.githubusercontent.com/97231749/193305440-1cc41ffe-cb40-4a5f-9d1b-b579ff58d241.png)

Reply:
Nope, that should not be possible for users outside IOHK group - as this would use the Buildkite resources (owned and paid by IOG);

in the future, we might add the option to run these tests on a free GitHub Actions Linux machine, but that would be for another time.